smartseq2_single_sample/check_reference/analysis/reference_check.ipynb

```diff
@@ -0,0 +1,561 @@
+{
```

Comment:
For (2), finalize gene biotypes included in reference:

I think the best thing to do here would be to lean on the experts at gencode and refseq. They seemed willing to help and are probably much better versed at identifying which biotypes we should see and which we shouldn't. 

A few things piqued my interest during the call: 
1. some small RNAs are detectable because they're carried within larger lincRNA. It sounds like at least refseq does not have any annotation to partition these from those that are directly transcribed as short hairpins and therefore are too small to detect. We should not deplete these. 
2. Transcribed pseudogenes are a biotype that I wasn't aware of. These could be included but would result in multi-mapping of those genes. We should investigate this further.

Reply:
For AC 0, when I work with Jupyter notebooks, I have two ways to check typos:
1. Just copy that whole markdown text you want to check manually into Grammarly(Free) version is enough, and copy back after fixing the typos.
2. There is a great spellchecker plugin help you check typos, install it by below comments:
```bash
pip install jupyter_contrib_nbextensions # install jupyter_contrib_nbextensions
jupyter contrib nbextension install --user # copy required javascript files
jupyter nbextension enable spellchecker/main # enable the spellcheck plugin
```
For more information, you could check [here](http://jupyter-contrib-nbextensions.readthedocs.io/en/latest/install.html#install-javascript-and-css-files)

Reply:
Ah cool; I already have nbextensions installed. That's very easy! Thanks @rexwangcc